lib/api.rb

```diff
@@ -11,7 +11,7 @@ module Api
   UnsupportedMediaTypeError = Class.new(ApiError)
 
   def self.encrypted_attribute?(attr)
-    Environment.encrypted_attributes.include?(attr.to_s) || attr.to_s.include?('password')
+    attr != "verify_ssl" && Environment.encrypted_attributes.any? { |a| attr.to_s.include?(a.to_s) }
```

Comment:
verify being a "weird" case is annoying.  I think I might change the UI to give a different name like password_verify, so we can get rid of verify from the filter list.